common/scripts/prep-samples.js

```diff
@@ -112,12 +113,21 @@ async function enableLocalRun(fileName, baseDir, pkgName) {
   const depth =
     relativeDir.length - relativeDir.split(path.sep).join("").length;
 
-  const relativeImportPath = new Array(depth).fill("..").join("/") + "/src";
+  let relativePath = new Array(depth).fill("..").join("/");
```

Comment:
nit: `path.sep` instead of `/`

Reply:
IIRC module paths always use `/` even on Windows,

Reply:
ah that's right